### ISSUE TITLE
UnsavedRelationList Error with ItemsToRender when no items to return

### ISSUE THREAD

**Issue by adrian-stein** [silverstripe/silverstripe-widgets#123]
I get the following error when there are no widgets to return

```
[User Error] Uncaught LogicException: filter can't be called on an UnsavedRelationList

Trace
UnsavedRelationList->filter(Enabled,1) 
WidgetArea.php:59
WidgetArea->ItemsToRender() 
WidgetArea.php:33
WidgetArea->WidgetControllers() 
ViewableData.php:402
```

Using SS 3.2.3 and widgets 1.2.*@dev

**Comment by tractorcow** [comment]
Fix in core at https://github.com/silverstripe/silverstripe-framework/pull/5348.

Sorry we probably can't fix this earlier than 3.4.0 because it would likely be considered "adding" api to non-feature releases.

**Comment by tractorcow** [comment]
Fixed with https://github.com/silverstripe/silverstripe-framework/pull/5348
3.4 will be coming out soon :D update to 3.x for now to get this fix.
